go-controller/pkg/ovn/policy.go

```diff
@@ -308,55 +313,35 @@ func getACLMatch(portGroupName, match string, policyType knet.PolicyType) string
 	return aclMatch
 }
 
-func namespacePortGroupACLName(namespace, portGroup, name string) string {
-	policyNamespace := namespace
-	if policyNamespace == "" {
-		policyNamespace = portGroup
-	}
-	if name == "" {
-		return policyNamespace
-
-	}
-	return fmt.Sprintf("%s_%s", policyNamespace, name)
+func getDefaultDenyPolicyACLName(ns, defaultDenySuffix string) string {
+	return joinACLName(ns, defaultDenySuffix)
 }
 
-func buildACL(namespace, portGroup, name, direction string, priority int, match, action string,
-	logLevels *ACLLoggingLevels, policyType knet.PolicyType) *nbdb.ACL {
-	var options map[string]string
-	aclName := namespacePortGroupACLName(namespace, portGroup, name)
-	var externalIds map[string]string
-	if policyType != "" {
-		externalIds = map[string]string{
-			defaultDenyPolicyTypeACLExtIdKey: string(policyType),
-		}
-	}
-	if policyType == knet.PolicyTypeEgress {
-		options = map[string]string{
-			"apply-after-lb": "true",
-		}
-	}
+func getDefaultDenyPolicyExternalIDs(aclType ACLType) map[string]string {
```

Comment:
Maybe we could rename this to `getDefaultPolicyExternalIDs`, since it is also use for multicast allow ACLs, and can potentially be reused for gress ACLs as well.

Reply:
yeah the fact that it is used for multicast is actually a problem, since id key explicitly says `defaultDenyPolicyTypeACLExtIdKey`, in the next PRs multicast will use its own ids, and these will only be used by default deny acls

Reply:
OK! I think my suggestion would work better in the interim though, just in case next PR take sits time, you never know :P
Up to you.

Reply:
oops, forget about this, I made the comment thinking that we used the same external id for gress policies.
We do set the policy type but under a different key.
If we plan on synchronizing all the ACLs, we might want to consolidate all the info we set on all ACLs with a common set of keys.